geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -7,7 +7,7 @@ Version 2.5.0
 Information
 -----------
 
-1.
+1. From now the basic auth is disabled by default.
```

Comment:
To enable basic auth see: https://camptocamp.github.io/c2cgeoportal/master/integrator/security.html#basic-auth

Reply:
+1 :-)